bofire/surrogates/diagnostics.py

```diff
@@ -625,7 +625,7 @@ class CvResults(BaseModel):
 
     # TODO[pydantic]: We couldn't refactor the `validator`, please replace it by `field_validator` manually.
```

Comment:
Can you remove this comment? It stems from the auto-conversion from pydantic 1 to pydantic 2 ;)